### ISSUE TITLE
Define architecture and features required for operational pipeline using PEcAn.

### ISSUE THREAD

**Issue by dlebauer** [ccmmf/organization#3]
**Description**

Definition of Done:
- requirements suitable for conceptual framework
- updated high level system architecture
- required features, including new features
- written into prose as part of the conceptual framework document

Out of scope:
- level of detail required for milestones 2.1 and 2.2: modeling framework

**Tasks**

- [ ] #18
- [x] Define requirements for operational pipeline
  - Core features
  - 'Nice to have'
  - User Interface
  - align requirements w/ existing PEcAn features
- [x] Identify new features that need to be developed

**Dependencies**

depends on #9 conceptual framework outline

**Additional Information**

- As defined in proposal ([Technical Portion](https://docs.google.com/document/d/1Vv5wP-UGOcbFTZ6XeYkkC9xOz3s5W9VprDI-SDdUsCU/edit#heading=h.41ybngyajbwv))
- will fit under task 2.1 in [conceptual framework](https://docs.google.com/document/d/1IApb-4m0en4ku_sclDDsWKjUXKuhQxLMFgmnt9e8zNk/edit)

**Comment by dlebauer** [comment]
Draft submitted; moved improved diagrams to new issue #18
